src/ripple/app/reporting/ETLSource.cpp

```diff
@@ -760,13 +760,24 @@ ETLLoadBalancer::forwardToP2p(RPC::JsonContext& context) const
     srand((unsigned)time(0));
     auto sourceIdx = rand() % sources_.size();
```

Comment:
Don't fix this right now, but I think there's a better way to get a uniform int distribution post C++11: https://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution  This may not be a critical place for it, but we should still model good coding practices.

That's a good change to make some time in the future.